### PR TITLE
 Add parameters to SemanticActions; implement extend selection for a11y

### PR DESCRIPTION
**Pull request by goderbauer**
Framework change in https://github.com/flutter/flutter/pull/13490.

Required for https://github.com/flutter/flutter/issues/13391.

### CODE REVIEW

File: shell/common/platform_view.cc [hunk 1]
```diff
@@ -55,12 +55,13 @@ void PlatformView::DispatchPlatformMessage(
...
-                                           blink::SemanticsAction action) {
+                                           blink::SemanticsAction action,
+                                           std::vector<uint8_t> args) {
   blink::Threads::UI()->PostTask(
-      [ engine = engine_->GetWeakPtr(), id, action ] {
+      [ engine = engine_->GetWeakPtr(), id, action, args = std::move(args) ] {
         if (engine) {
           engine->DispatchSemanticsAction(
-              id, static_cast<blink::SemanticsAction>(action));
+              id, static_cast<blink::SemanticsAction>(action), args);
```
Comment (jason-simmons) [inline shell/common/platform_view.cc]: pass std::move(args) here (and in Engine and RuntimeController)

**Comment by zanderso** [comment]
It looks like this commit causes a failure when running Flutter gallery:
```
E/flutter ( 3611): type '(int, SemanticsAction) => void' is not a subtype of type 'SemanticsActionCallback' of 'callback'
E/flutter ( 3611): #0      Window.onSemanticsAction= (file:///usr/local/google/home/zra/flutter/engine/src/flutter/lib/ui/window.dart:601)
E/flutter ( 3611): #1      BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&RendererBinding.initInstances (package:flutter/src/rendering/binding.dart:41)
E/flutter ( 3611): #2      BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&RendererBinding&WidgetsBinding.initInstances (package:flutter/src/widgets/binding.dart:241)
E/flutter ( 3611): #3      new BindingBase (package:flutter/src/foundation/binding.dart:51)
E/flutter ( 3611): #4      new BindingBase&GestureBinding (package:flutter/src/gestures/binding.dart:19)
E/flutter ( 3611): #5      new BindingBase&GestureBinding&ServicesBinding (package:flutter/src/services/binding.dart:19)
E/flutter ( 3611): #6      new BindingBase&GestureBinding&ServicesBinding&SchedulerBinding (package:flutter/src/scheduler/binding.dart:168)
E/flutter ( 3611): #7      new BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding (package:flutter/src/painting/binding.dart:15)
E/flutter ( 3611): #8      new BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&RendererBinding (package:flutter/src/rendering/binding.dart:23)
E/flutter ( 3611): #9      new BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&RendererBinding&WidgetsBinding (package:flutter/src/widgets/binding.dart:234)
E/flutter ( 3611): #10     new WidgetsFlutterBinding (package:flutter/src/widgets/binding.dart:878)
E/flutter ( 3611): #11     WidgetsFlutterBinding.ensureInitialized (package:flutter/src/widgets/binding.dart:893)
E/flutter ( 3611): #12     runApp (package:flutter/src/widgets/binding.dart:688)
E/flutter ( 3611): #13     main (file:///usr/local/google/home/zra/flutter/flutter/examples/flutter_gallery/lib/main.dart:13)
E/flutter ( 3611): #14     _startIsolate.<anonymous closure> (dart:isolate-patch/dart:isolate/isolate_patch.dart:277)
E/flutter ( 3611): #15     _RawReceivePortImpl._handleMessage (dart:isolate-patch/dart:isolate/isolate_patch.dart:163)
```

**Comment by goderbauer** [comment]
@zanderso You will need this framework-side change: https://github.com/flutter/flutter/pull/13490

It's ready to commit once the bots have build the new engine version with this change.

**Comment by goderbauer** [comment]
Looks like the bots are failing to build the engine with this change. Rolling this back for now until I know what's going on.

**Pull request merged**